Makefile

```
@@ -0,0 +1,23 @@
+export SHELL := /bin/bash
+export SHELLOPTS := errexit
+
+GOPATH ?= $(shell go env GOPATH)
+BIN_DIR := $(GOPATH)/bin
+GOLANGCI_LINT := $(BIN_DIR)/golangci-lint
+
+.PHONY: lint lintfix test build
+
+$(GOLANGCI_LINT):
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(BIN_DIR) v1.54.2
```

This link doesn't work.
```suggestion
	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BIN_DIR) v1.54.2
```

---

Thanks